test/integration/reset_two_step_verification_test.rb

```diff
@@ -0,0 +1,109 @@
+require "test_helper"
+
+class ResetTwoStepVerificationTest < ActionDispatch::IntegrationTest
```

Comment:
Should we also be testing that users can setup 2sv? I think from the change above it looks that it is also an affect of this change

Reply:
Good point @kevindew, are you thinking of e.g. [this test](https://github.com/alphagov/signon/blob/main/test/integration/super_admin_flagging_two_step_verification_test.rb)

Reply:
I'd wonder if it could be part of this test somehow. Perhaps it could go to each user ask them to set-up 2sv and then reset it to do both actions?

Reply:
Cool, I'll have a look. Good suggestion too - while testing this it turns out that other admin users _cannot_ set 2sv on users, so good shout

Reply:
Have updated - I didn't end up putting it in the same test because it would have become a more involved test requiring us to log in as the user to set up 2sv in the middle, before we could reset it. But yes, adding the tests uncovered that other kinds of admin users couldn't actually update the 2sv requirement, so have definitely caught a bug that would have at some point been raised on zendesk.